### PR TITLE
Fix filterM comment

### PR DESCRIPTION
**Pull request by totem3**
Just update comment. filterM comment is same as mapM.

### CODE REVIEW

File: src/Data/Set/Monad.hs [hunk 1]
```diff
@@ -22,7 +22,7 @@ mapM_ :: (Ord a,Ord b,Monad m)
        => (a -> m b) -> Set a -> m ()
 mapM_ f = L.mapM_ f . S.toList
 
--- | Map over a 'Set' in a monad.
-filterM :: (Ord a,Monad m)
+-- | Filter elements of a 'Set' in a monad.
+filtekrM :: (Ord a,Monad m)
```
Comment (snoyberg) [inline src/Data/Set/Monad.hs]: I think that `k` is unintentional :)

**Comment by chrisdone** [comment]
This breaks the code.  Please check your commit.

**Comment by snoyberg** [comment]
Thanks!

**Comment by totem3** [comment]
oops, sorry :(  i fixed that

**Pull request merged**